src/elements/content-explorer/MetadataViewContainer.tsx

```diff
@@ -80,60 +117,181 @@ const MetadataViewContainer = ({
     actionBarProps,
     columns,
     currentCollection,
+    fieldsToShow,
```

Comment:
why is there both `columns` and `fieldsToShow`? what's the difference between the two and how are consumers using the two? i would've expected the consumer just passes the columns prop